### ISSUE TITLE
suricate server to be stopped before starting up the  LPContainer 

### ISSUE THREAD

**Issue by jjpopski** [marco-buttu/suricate#74]
If suricate server is running and the LP Receiver is being monitored, the LPBand Container crashes when started.  To make the container run, suricate server is to be stopped. 

```
Starting Container...
DALReference obtained via command line: 'corbaloc::discos-manager.srt.inaf.it:3012/CDB'
ContainerStatusMsg: Startup begins
ContainerStatusMsg: ORB initialization begins
ContainerStatusMsg: ORB initialization ends
ContainerStatusMsg: Manager access initialization begins
ContainerStatusMsg: Manager access initialization ends
ContainerStatusMsg: Startup ends
2020-01-07T16:29:56.015 [SRTLPBandContainer-GL - ] Alarm sent: <BACIProperty,SRTLPBandReceiver,2> ACTIVE
2020-01-07T16:29:56.016 [SRTLPBandContainer-GL - ] Alarm sent: <BACIProperty,SRTLPBandReceiver,2> ACTIVE
2020-01-07T16:29:56.016 [SRTLPBandContainer-GL - ] Alarm sent: <BACIProperty,SRTLPBandReceiver,1> ACTIVE
2020-01-07T16:29:56.016 [SRTLPBandContainer-GL - ] Alarm sent: <BACIProperty,SRTLPBandReceiver,1> ACTIVE
2020-01-07T16:29:56.016 [SRTLPBandContainer-GL - ] Alarm sent: <BACIProperty,SRTLPBandReceiver,1> ACTIVE
ContainerStatusMsg: Automatic component loading begins
ContainerStatusMsg: Automatic component loading ends
ContainerStatusMsg: Ready
/alma/ACS-FEB2017/ACSSW/bin/acsStartContainer: line 112: 62232 Segmentation fault      (core dumped) $COMMANDLINE
2020-01-07T16:30:01.989 INFO [acsStartContainer] Container: 'SRTLPBandContainer' exited with code: 139.
```

**Comment by marco-buttu** [comment]
I reproduced the issue (with a receiver board simulator `receiverboard-sim`):

* Suricate is running and monitoring the LP component
* The LP component is running
* The LP component is active on `objext`
* Using `objexp` ask for a LP property value (not sure it is required to spot the issue)
* Close the LP container
* Using `objexp` (not a new instance but the previous `objexp` instance) ask for a LP property value: it will raise a CORBA.TRANSIENT exception
* Restart the LP container: **crash after a few seconds!**

There is something wrong in the component, but maybe I can avoid the issue by asking (with Suricate) for a new component every time there is a CORBA error. Also look at how I am releasing the component (with Suricate).

**Comment by marco-buttu** [comment]
For the Suricate point of view, the issue has to be fixed as described in the last part of [this message]( https://github.com/discos/discos/issues/524#issuecomment-597607410).

**Comment by marco-buttu** [comment]
@jjpopski I think we can close this issue because finally I spot the real problem: something wrong at the LPBand component startup, as explained in [this comment](https://github.com/discos/discos/issues/524#issuecomment-597607410). Until the [LP component issue](https://github.com/discos/discos/issues/524) is still open, I suggest to bypass the LP component misbehavior just avoiding to monitor the `vd`, `id` and `vg` properties.

**Comment by marco-buttu** [comment]
As explained in [this comment](https://github.com/discos/discos/issues/524#issuecomment-597607410), there is no issue the monitoring of the `vd`, `vg` and `id` properties  starts at least 5 seconds after getting the component. To avoid similar issues, for every component I put a `STARTUP_WTIME` (startup waiting time) in the configuration file: the monitoring of the properties will not start at right after component startup, but `STARTUP_WTIME` seconds later.

**Comment by marco-buttu** [comment]
Fixed in commit bf035fc.
